contrib/kind.sh

```diff
@@ -409,6 +414,27 @@ if ! kubectl wait -n kube-system --for=condition=ready pods --all --timeout=300s
   kubectl get pods -A -o wide || true
   exit 1
 fi
+if [ "$OVN_MONITOR" == true ]; then
+  # Begin by creating the monitoring namespace and prometheus CRDs
+  for file in `curl  -L https://api.github.com/repos/prometheus-operator/kube-prometheus/contents/manifests/setup | jq '.[].download_url'`; do
+    run_kubectl apply -f $file
+  done
+  # Then get all kube-state-metrics, node-exporter and prometheus templates, and deploy them
+  for file in `curl  -L https://api.github.com/repos/prometheus-operator/kube-prometheus/contents/manifests | jq '.[] | select(.name|test("kube.|node.|prometheus-.")) | select(.name|test("prometheus-adapter.")|not)| .download_url'`; do 
+    run_kubectl apply -f $file
+  done
+  if ! kubectl wait -n monitoring --for=condition=ready pods --all --timeout=300s ; then
+    echo "some pods in the monitoring namespace are not running"
+    kubectl get pods -n monitoring -o wide || true
+    exit 1
+  fi
+  pushd ../dist/yaml
+  run_kubectl apply -f ovnkube-monitor.yaml
+  popd
+  echo "Pods are all up, your Prometheus instance is now accessible on localhost:9090"
+  # Expose the prometheus service so that we can access it from localhost:9090
+  exec kubectl -n monitoring port-forward svc/prometheus-k8s 9090
```

Comment:
As `port-forward` doesn't return, It might be nicer to echo out the command instead.

Alternatively, could we expose this [using a NodePort](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/exposing-prometheus-and-alertmanager.md#nodeport) instead

Reply:
OK, I will just echo it out. Using a NodePort service requires extra configuration from the developer, as the KIND deployment runs on a separate docker network, which will require the developer to setup routes if it's to be accessed from his/hers web browser.